pkg/install/v1alpha1/openebs_crds.go

```diff
@@ -291,15 +291,15 @@ spec:
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: backupcstors.openebs.io
+  name: cstorbackups.openebs.io
 spec:
   group: openebs.io
   version: v1alpha1
   scope: Namespaced
   names:
-    plural: backupcstors
-    singular: backupcstor
-    kind: BackupCStor
+    plural: cstorbackups
+    singular: cstorbackup
+    kind: CStorBackup
     shortNames:
     - bkp
```

Comment:
can this be `cbkp`

Reply:
ok. I will add prefix `c` to short names to avoid any conflict with velero/ark CRD.